test/sharness/t0080-repo.sh

```diff
@@ -98,7 +104,7 @@ test_expect_success "remove recursive pin, add direct" '
 
 test_expect_success "remove direct pin" '
 	echo "unpinned $HASH" >expected6 &&
-	ipfs pin rm "$HASH" >actual6 &&
+	ipfs pin rm -r=false "$HASH" >actual6 &&
```

Comment:
But then there is no way to remove only recursive pins. -r will remove direct pins as well. Is that intended way for -r to behave?